fuzz/fuzz_targets/differential.rs

```diff
@@ -35,16 +39,22 @@ fuzz_target!(|data: &[u8]| {
         // `setup_ocaml_runtime`.
         engine::setup_engine_runtimes();
 
+        let (default_engines, default_modules) = if cfg!(feature = "fuzz-winch") {
+            (vec!["wasmi"], vec!["wasm-smith", "single-inst"])
+        } else {
+            (
+                vec!["wasmtime", "wasmi", "spec", "v8"],
+                vec!["wasm-smith", "single-inst"],
+            )
+        };
+
         // Retrieve the configuration for this fuzz target from `ALLOWED_*`
         // environment variables.
-        let allowed_engines = build_allowed_env_list(
-            parse_env_list("ALLOWED_ENGINES"),
-            &["wasmtime", "wasmi", "spec", "v8"],
-        );
-        let allowed_modules = build_allowed_env_list(
-            parse_env_list("ALLOWED_MODULES"),
-            &["wasm-smith", "single-inst"],
-        );
+        let allowed_engines =
+            build_allowed_env_list(parse_env_list("ALLOWED_ENGINES"), &default_engines);
+        let allowed_modules =
+            build_allowed_env_list(parse_env_list("ALLOWED_MODULES"), &default_modules);
```

Comment:
How come winch changes the selection of default engines?

Reply:
To add a bit, @alexcrichton the intent of the design here is to give temporary support for Winch bringup; as @saulecabrera noted we hope that eventually it's "just another engine in the list". The alternative to this would be to develop a completely separate "bringup fuzz target" doing the targeted comparison that we want, which we decided would be less ideal (code duplication etc).

Reply:
That all makes sense to me, but for fuzzing locally the env vars can be set, so I'm not sure why this change was needed? It's not wrong of course, and I agree that there's no need for a whole separate fuzz target for winch. I was expecting, though, that how to fuzz winch would be "probably set some env vars to specific values to avoid known crashes and get quicker test coverage", so that way fuzzing winch in full is "remove the env vars" or similar.